AustinKellar/PowerOn-Prototype
# Player no longer gets stuck on branches and trunks

Note: This PR is intended to be merged after the style guide PR

Goal: Fix a bug where the player gets stuck on branches and trunks while in the air and moving horizontally

What I changed:
- Added a new Player object and deleted the old one from the system. I tried to make the new player 
feel almost the same as the old one
- The new player object also has a new Script attached to it called PlayerMovement.cs, which allows us to more finely cusomize the movement
- I needed to re-create the Branch prefab, so they are now dark brown and skinnier, with new stats
- I changed the collider on the sap objects to a trigger collider, so the player doesn't stagger when they hit it
- Added a PlatformEffector component to the tree trunks to keep the player from getting stuck to them
- Doubled the maxSap value in BranchSpawner.cs to keep up with the new branches

How to Test:
- Load up the game in the new branch
- Test player movement. Does it feel as good as the old character? 
- Try to get stuck on the branches and the tree trunks
- Ensure that the player does not stagger when it hits sap objects

Note:
- For now, I left the Asset for the old character object (~190 files of sprites, animations, etc) in the game to keep this PR to small. Should I delete it?
